inventory/osinfo/osinfo_linux.go

```diff
@@ -29,12 +28,11 @@ import (
 
 var (
 	entRelVerRgx = regexp.MustCompile(`\d+(\.\d+)?(\.\d+)?`)
-	getUname     = func() ([]byte, error) {
+
+	// GetUname returns the kernel details
+	GetUname = func() ([]byte, error) {
```

Comment:
Why is this public?

Reply:
i am stubbing the getuname function in inventory package

Reply:
made this private